pkg/credentialprovider/apis/credentials/BUILD

```diff
@@ -0,0 +1,36 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
add an OWNERS file to `pkg/credentialprovider/apis`:

```
# See the OWNERS docs at https://go.k8s.io/owners

# Disable inheritance as this is an api owners file
options:
  no_parent_owners: true
approvers:
- api-approvers
reviewers:
- sig-node-api-reviewers
- sig-auth-api-reviewers
```

Reply:
or move to `staging/src/k8s.io/kubelet/pkg/apis/credentialprovider` and put that OWNERS file under `staging/src/k8s.io/kubelet/pkg/apis`